<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>534</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>高速列车车轮组件异常动态智能检测系统</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0">
   <property name="leftMargin">
    <number>100</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>100</number>
   </property>
   <property name="bottomMargin">
    <number>100</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,2">
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="pic_high">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_title">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>高速列车车轮组件异常动态智能检测系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1,0,0,1,0,4">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>50</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QLabel" name="label_account">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_account">
             <property name="font">
              <font>
               <family>等线</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>请输入账号</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>50</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QLabel" name="label_pas">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_pas">
             <property name="font">
              <font>
               <family>等线</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>请输入密码</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>50</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QPushButton" name="button_login">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>等线</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>登录</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QLabel" name="info_coor">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="text">
              <string>研制单位
	安徽富煌钢构股份公司
	安徽君达高科公司
	安徽富煌科技股份公司
	北京交通大学
	安徽大学</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
